examples/webgl_tonemapping.html

```diff
@@ -67,7 +69,7 @@
 				document.body.appendChild( renderer.domElement );
 
 				renderer.toneMapping = toneMappingOptions[ params.toneMapping ];
-				renderer.toneMappingExposure = params.exposure;
+				renderer.toneMappingExposure = Math.pow( 10, params.logExposure );
```

Comment:
I do not think this re-parameterization of exposure is necessary. 

First, we do not want the user to set exposure to 10 -- much less 100. Such values are way over-bright.

Second, we may change exposure units to EVs, which is power-of-two based. But we have not had that discussion yet.

Reply:
Sure, I'll change it back. I needed it for the comparison images and figured others would want to be able to reproduce them